src/applications/vaos/components/BackLink.jsx

```diff
@@ -74,12 +52,18 @@ export default function BackLink({ appointment }) {
         ‹
       </div>
       <NavLink
-        aria-label={handleBackLinkText()}
-        to={handleBackLink()}
+        aria-label={linkText}
+        to={link}
         className="vaos-hide-for-print vads-u-color--link-default"
-        onClick={handleClickGATracker()}
+        onClick={() => {
+          recordEvent({
+            event: `${GA_PREFIX}-${status}-${progress}-details-descriptive-back-link-clicked`,
```

Comment:
Looking at the previous logic there's a piece (lines 32-33) that overrides status to be `direct` for `progress === 'confirmation' && status === 'upcoming'`. I think we still need that here right?

Reply:
Good eye. Putting logic back in.